javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -47,31 +44,31 @@ const BasicCourseTable = ( {classes} ) => {
     },{
       dataField: 'section',
       text: 'Section',
-      align: (cell, row) => dataAlignment(cell, row)
+      align: 'left'
```

Comment:
Do we even need the `align` field anymore? I think the cells default to left alignment, so if that's the case, we can even remove this as well. 

Try it out and see if it still produces the same effect. If it does, we should remove it; if not, let me know and we can keep it in.